src/ui/backend/BaseBacking.js

```diff
@@ -36,12 +36,21 @@ export default class BaseBacking {
     this.scaleY = 1;
     this.flipX = false;
     this.flipY = false;
-    this.visible = true;
+    this._visible = true;
     this.clip = false;
     this.backgroundColor = '';
     this.compositeOperation = '';
   }
 
+  get visible () { return this._visible; }
+  set visible (visible) {
+    if (this._visible === visible) {
+      return;
+    }
+    this._visible = visible;
+    this._onVisible();
```

Comment:
It looks like this is only implemented on the inheriting class, but it should have a base implementation too even if an empty function.